CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## master/main
```

Comment:
@cnlg-lanl You can make these changes `## 0.4.0` and `## master/main` will be blank, waiting for content from the next PRs.  Thank you!